tests/unit/parser/test_manifest.py

```diff
@@ -20,9 +25,25 @@ def mock_project():
     mock_project.profile_env_vars = {}
     mock_project.project_target_path = "mock_target_path"
     mock_project.credentials = MagicMock()
+    mock_project.clear_dependencies = MagicMock()
     return mock_project
 
 
+@pytest.fixture
```

Comment:
Shoude this live somewhere in the utils being used by other unittests?

Reply:
We can lift these 👍